apps/playground/src/app/components/forms/radio/radio.component.html

```diff
@@ -1,3 +1,4 @@
+<sky-radio />ß
```

Comment:
Did you mean to leave this random character there?

Reply:
nope!